src/internal/hooks/useLocalStorage.ts

```diff
@@ -70,7 +66,8 @@ export const useLocalStorage = (
   };
 
   return {
-    clearLocalStorage,
+    clearThemeLocalStorage,
```

Comment:
Along the lines of my hook separation comment above, maybe we should do the same for useLocalStorage. The paradigm is always
at root: /x/commonThing
/x/subItem1/item1SpecificThing
/x/subItem2/item2SpecificThing